build.gradle

```diff
@@ -19,7 +19,7 @@ compileTestJava { options.encoding = "UTF-8" }
 //    }
 //}
 
-version = "3.6.1"
+version = "3.6.2"
```

Comment:
This needs to change too....

Reply:
Done, didnt realise the file was unstaged